### PR TITLE
Average goals: added get_scores helper method and rspec

### PR DESCRIPTION
**Pull request by ejdesh**
Stattracker
  #get_scores
    returns an array of all scores for a team_id

Finished in 0.00351 seconds (files took 0.29283 seconds to load)
1 example, 0 failures

Coverage report generated for RSpec to /Users/eric/projects/turing/mod1/futbol/coverage.
Line Coverage: 92.06% (116 / 126)

### CODE REVIEW

File: stat_tracker.rb
Comment (Crosswolfv1) [inline stat_tracker.rb:1]: thank you for removing this file

File: spec/stat_tracker_spec.rb
Comment (Crosswolfv1) [inline spec/stat_tracker_spec.rb:1]: good helper methods, and thanks for fixing some formatting issues

File: runner.rb
Comment (Crosswolfv1) [inline runner.rb:1]: prob don't need this to be branch merged, recommend add to your gitignore

**Review by Crosswolfv1 — CHANGES_REQUESTED** [review]
We should discuss about having a global runner.rb so we are on the same track

**Review by Crosswolfv1 — APPROVED** [review]

File: runner.rb
Comment (Crosswolfv1) [reply runner.rb:1]: Discussed runner file in a quick standup, we are clear now

**Comment by Montana-Pfeifer** [comment]
methods look good and test fulfill all  requirements good to merge.

**Pull request merged**